test/soca/gw/setup_obsprep.sh

```diff
@@ -13,6 +13,11 @@ project_source_dir="$1"
 testdatadir="${project_source_dir}/test/soca/testdata"
 testdatadir_bufr="${project_source_dir}/build/gdas/test/testdata"
 
+# run the g-w link script so that the config files can be found
+cd ${project_source_dir}/../../..
```

Comment:
why not just 
```bash
(cd ${project_source_dir}/../../.. && ./link_workflow.sh)
```
?

Reply:
Also, can you not do:
`${project_source_dir}/../../../link_workflow.sh` ?

the script should get its path regardless of where it is executed from

Reply:
@guillaumevernieres Your bash-fu is superior to mine, Sir
@CoryMartin-NOAA I tried that and for some reason it didn't like it